corehq/apps/data_vault/utils.py

```diff
@@ -0,0 +1,3 @@
+def _get_tracked_vault_entries(on_model):
```

Comment:
I think you should drop the leading `_` since this is used outside of the module. Or you could keep the `_` and move it to the package `__init__` where it's used.

Reply:
the intention was to convey that this should not be used outside this app. Is there a better way to do that or is that already clear because its not in `__init__`? The reason to move it to a separate file was because I didn't want to clutter the `__init__` file and keep only the methods we would actually want to expose outside the app in it.

Reply:
By importing into `__init__` it's effectively the same as defining it there from the point of view of a caller. The convention for `_` methods is that they are module or class scoped. I don't think there's a convention for 'package' scoped function naming. Another way to indicate what is the public interface of a package is to use the `__all__` variable:
```python
__all__ = ("funcaion_a", "funcaion_b")
```

Reply:
thank you for the detailed answer @snopoke 
https://github.com/dimagi/commcare-hq/pull/28161/commits/dba63e70cf230afc5a5ed4d91a664c99c32d2e8d